lib/travis/api/v3/services/env_var/delete.rb

```diff
@@ -0,0 +1,12 @@
+module Travis::API::V3
+  class Services::EnvVar::Delete < Service
+    params :id, prefix: :repository
+    params :id, prefix: :env_var
+
+    def run!
+      raise LoginRequired unless access_control.logged_in? or access_control.full_access?
```

Comment:
I think you're right – will change this to `||`. Also a great idea to extract this into `Service`.